presto-main/src/main/java/com/facebook/presto/operator/scalar/tdigest/TDigestUtils.java

```diff
@@ -328,7 +416,8 @@ private static void quickSort(double[] key, double[][] values, int start, int en
      * @param limit  The largest amount of disorder
      */
     @SuppressWarnings("SameParameterValue")
-    private static void insertionSort(double[] key, double[][] values, int start, int end, int limit) {
+    private static void insertionSort(double[] key, double[][] values, int start, int end, int limit)
```

Comment:
Do we use this?

Reply:
Yes, it's used in another function in TDigestUtils, 
```
public static void sort(double[] key, int start, int n, double[]... values)
```
And `sort` is used in `TDigest` class